code/modules/uplink/uplink_items/uplink_clothing.dm

```diff
@@ -21,26 +21,15 @@
 	cost = 1
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops) //They already get these
 
-/datum/uplink_item/suits/padding
-	name = "Soft Padding"
-	desc = "An inconspicious soft padding meant to be worn underneath jumpsuits, will cushion the user from melee harm."
-	item = /obj/item/clothing/accessory/padding
-	cost = 2
-	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
-
-/datum/uplink_item/suits/kevlar
-	name = "Kevlar Padding"
-	desc = "An inconspicious kevlar padding meant to be worn underneath jumpsuits, will cushion the wearer from ballistic harm."
-	item = /obj/item/clothing/accessory/kevlar
-	cost = 2
-	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
-
-/datum/uplink_item/suits/plastic
-	name = "Ablative Padding"
-	desc = "An inconspicious ablative padding meant to be worn underneath jumpsuits, will cushion the wearer from energy lasers harm."
-	item = /obj/item/clothing/accessory/plastics
-	cost = 2
-	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+/datum/uplink_item/suits/infiltrator_bundle
+	name = "Insidious Infiltration Gear Case"
+	desc = "Developed by Roseus Galactic in conjunction with the Gorlex Marauders to produce a functional suit for urban operations, \
+			this suit proves to be cheaper than your standard issue hardsuit, with none of the movement restrictions of the outdated spacesuits employed by the company even if it is still quite yet approved for deep space infiltrations. \
+			Comes with an armored vest, helmet, blood-red sneaksuit, sneakboots, specialized combat gloves and a high-tech balaclava which obfuscates both your voice and your face. The case is also rather useful as a storage container and bludgeoning implement."
+	item = /obj/item/storage/toolbox/infiltrator
+	cost = 3
```

Comment:
I dont get why you get well over 30 tcs worth of gear for 3 tc...

Reply:
the case is a normal sized toolbox that can only hold two very specific firearms and ammunition
the helmet is essentially a space suit helmet with a bit of armor at the cost of no space protection (you can literally spear open customs for equivalents)
the vest is essentially a security vest with more flair for the fact you get instaraped for wearing it, again, raid customs for the exact same thing essentially but with less suspiciousness
the undersuit is probably the only unique piece of armor you get out of it, tens across the board which can be sort of matched by security jumpsuits, which once again you can get ten plus of them from customs and or buy the tactical turtleneck for one singular telecrystal.
the gloves are essentially insulated nitrile gloves which makes them a bit more pricy than the ones you get out of the syndicate toolbox
the boots are combat boots so you get some nice armor to your legs but who is aiming for your legs and have the very minor buff of being less vulnerable to people with their audio up to maximum and the ear training to hear footsteps
and finally the balaclava is an explorer gas mask with the feature that you are always unknown when talking which can pretty easily be accomplished to an arguably much better degree by the chameleon mask

i fail to see how this is worth more than thirty telecrystals, i might bump it up to four because even at three it is a really good deal, but for the love of god this shit is not more than six at all unless it had deathsquad tier resistances and space immunity at all times
you can physically get gear on par with it by raiding the customs outpost with a spear
how the fudge

i fail to see how this is worth more than thirty telecrystals, i might bump it up to four because even at three it is a really good deal, but for the love of god this shit is not more than six at all unless it had deathsquad tier resistances and space immunity at all times
you can physically get gear on par with it by raiding the customs outpost with a spear

Reply:
Lets go through its items as you have, gloves - Speedy bios that are amazing at not dieing to fire and shocks, should prob be free to traitors or at lest cost 1 for "muh balance", The mask is the voice changer one, meaning it would normally cost quite a bit more then just 1, but is rather obs and no real masked stealth without a helm that covers mask. So lets say 2 tc.
The box is a kinda good melee weapon I guess like the normal tool box, so 1 tc.
Combat boots are amazing 🗡 1tc should /prob/ be normal traitor gear for 1 tc imo as well.
The suit is a better syndi suit so 2tc cuz it has some p-good armor on it.
helm is kinda trash no one really aims head these days when its all about that slowdown in the legs.

Reply:
once again, the gloves are combined nitriles and combat gloves, except **wearing them makes you ultra valid** worth about one telecrystal
the mask is a voice changer without the actual good part of being able to change anything, if anyone sees someone talking as exclusively unknown and sees them wearing a balaclava, **you once again are ultra valid** worth about maybe half to one telecrsyal, the chameleon kit one does its function but better, and there you get all of the other stealthy chameleon kit gear too
the case is literally **a toolbox in force that makes you valid** are you seeing a trend? all of these items have the nice features of all the powergame gear except they make you super duper ultra valid salad when wearing them, nobody is going to question someone wearing nitrile gloves, and if you have combat gloves you can claim you got them from lavaland and or deep space, but if you have insidious combat gloves the one and only explanation is cargo has null crates, and or you are a traitor
the combat boots are combat boots, which once again can be gotten from deep space, **and who the fuck aims for the legs** when the most effective piece to be aiming at if you want to induce stamcrit is the chest since legs stop taking damage once their cap has been reached and they are disabled
the suit is better than the tactical turtleneck, yes, but rather than being very slightly inconspicuous to the untrained eye since anyone can misread tactical as tacticool quite easily it is bright red, obvious as fuck, and essentially a syndie balloon, worth about one telecrystal, ish

yes, these things are all a very good deal for three telecrystals, but the down side is if you are using any of these things anyone on your screen is going to immediately assume you are valid salad, since you are wearing a bright red very nukie esque combat suit, security and valid hunting assistants alike are going to get bulges in their pants at the thought of taking your sick gamer gear

Reply:
Pretty much this to a T. While all of this carries very obvious advantages that are handy in their own ways, the suit is entirely conspicuous to just about anyone who sees it and already know to assume 'blood red man bad'.

Traitor equipment is largely inconspicuous or might go overlooked by someone who isn't looking for the specific cues that indicate to them that this person intends harm until it's too late. Wearing the Infiltrator kit is basically forgoing that entirely to go loud as fuck in a very brash manner.

It's only really going to be useful if you're constantly hit and running and alpha striking, in which case its colour palette and sound cancelling will allow for you to be far harder to track in maint, as an example.